tests/constructors.rs

```diff
@@ -14,18 +14,22 @@ impl Deref for StringLike<'_> {
 
 #[test]
 fn main() {
-    let s = "";
-    let _ = TextSize::of(&s);
-
-    let s = String::new();
-    let _ = TextSize::of(&s);
-
-    let s = Cow::Borrowed("");
-    let _ = TextSize::of(&s);
-
-    let s = Cow::Owned(String::new());
-    let _ = TextSize::of(&s);
+    macro_rules! test {
+        ($($expr:expr),+ $(,)?) => {
+            $({
+                let s = $expr;
+                let _ = TextSize::of(&s);
+            })+
+        };
+    }
 
-    let s = StringLike("");
-    let _ = TextSize::of(&s);
+    test! {
+        "",
```

Comment:
Do we test `::of("")` (without &) anywhere?

Reply:
In the doc-tests for `TextSize::of`, yes, but I'm going to add it here as well.